service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/FeedbackReceiver.java

```diff
@@ -36,7 +36,7 @@
      * @param iotHubServiceClientProtocol The iot hub protocol name
      * @param deviceId The device id
      */
-    public @Deprecated FeedbackReceiver(String hostName, String userName, String sasToken, IotHubServiceClientProtocol iotHubServiceClientProtocol, String deviceId)
+    public @Deprecated FeedbackReceiver(String hostName, String userName, String sasToken, IotHubServiceClientProtocol iotHubServiceClientProtocol, String deviceId, ProxyOptions proxyOptions)
```

Comment:
Why are we adding support to deprecated API?

Reply:
Good question, give me a sec to check

Reply:
I can roll back this change actually. I'll just need to throw an unsupportedOperationException if the user tries to make proxy support work with these deprecated APIs